pkg/scheduler/factory/signal.go

```diff
@@ -0,0 +1,25 @@
+// +build !windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package factory
+
+import "syscall"
+
+// compareSignal is the signal to trigger cache compare. For non-windows
+// environment it's SIGUSR2.
+var compareSignal = syscall.SIGUSR2
```

Comment:
I think we will enable this in test-infra, how will test-infra expect to trigger it may I ask?

Reply:
you mean test this in e2e test? no, it's for whoever interested in triggering the comparison to debug. Nothing would happen except some logs are appended.

Reply:
I saw it was mentioned in the issue description: 
> We will probably enable it in our tests and in clusters where customers see stale cache issues

Anyway this question will not block this PR. I will re-lgtm it when CI passed.

Reply:
@resouer, Your point is valid. In fact, I also chatted with @anfernee on Slack about periodic triggering of this or enabling it in tests. I believe we should try this feature and make sure that we do not see a lot of false positives. We should also measure its performance overhead in larger tests to ensure that enabling it in tests won't cause timeouts. Once it is refined, we may want to enable it in certain tests.

Reply:
Got it, thanks for explaining.